Optik/Classes/ImageViewController.swift

```diff
@@ -30,7 +30,8 @@ internal final class ImageViewController: UIViewController {
         }
     }
     private(set) var imageView: UIImageView?
-
+    var likeButton: UIButton?
```

Comment:
I'm not talking about var, I'm talking about how it's created and handled. Why is it different?